baserun/helpers.py

```diff
@@ -14,4 +19,28 @@ class BaserunType(Enum):
 class BaserunStepType(Enum):
     LOG = auto()
     AUTO_LLM = auto()
-    CUSTOM_LLM = auto()
\ No newline at end of file
+    CUSTOM_LLM = auto()
+
+
+def baserun_threaded_wrapper(
+    target: Callable, results: list, thread_args=None, thread_kwargs=None
```

Comment:
Is it possible to leave the signature as:

`baserun_threaded_wrapper(target: Callable)` and basically give wrapper the args / kwargs parameters which will be passed through when invoked by the threading module?